src/subscriber.ts

```diff
@@ -1,32 +1,39 @@
 import { ApiPromise, WsProvider } from '@polkadot/api';
-import { SessionIndex } from '@polkadot/types/interfaces';
+import { SessionIndex, Registration, IdentityInfo } from '@polkadot/types/interfaces';
 import { Logger } from '@w3f/logger';
 import { Text } from '@polkadot/types/primitive';
-
 import {
-    InputConfig
+    InputConfig, JudgementRequest, StorageData
 } from './types';
+import Event from '@polkadot/types/generic/Event';
+import { Option } from '@polkadot/types'
+import fs from 'fs'
+import storage from 'node-persist';
 
 export class Subscriber {
     private chain: Text;
     private api: ApiPromise;
     private endpoint: string;
     private sessionIndex: SessionIndex;
     private logLevel: string;
+    private readonly PERSISTENCE_DIR = './persistence'
```

Comment:
We need to modify the chart, this directory should be a persistent volume and we should use a statefulset instead of a deployment

Reply:
Unfortunately the so convenient method api.query.identity.identityOf() with no arguments seems not exists in the library. It always require a parameter, and therefore it's not possible to trigger it without parameters. I guess from the ui something happens under the hood if you toggle the flag 'include option'.

I dived into the code of the api and the ui but for now I prefer to give up and focus on providing a working solution, wdyt ?

Reply:
Maybe a quick question to Jaco can save a lot of time. If it's not possible to use the onchain data let's go ahead with offchain persistence for a working solution.